suites/squid/nvmeof/tier-3_8-1-nvmeof_4-group_8-gw_scale.yaml

```diff
@@ -0,0 +1,518 @@
+          - config:
+              command: add_host
+              args:
+                subsystems: 76
+                namespaces: 1520
+                initiators:
+                  - node41
+                  - node42
+                  - node43
+                  - node44
+                  - node45
+                group: group1
+        initiators:
+          - node41
+          - node42
+          - node43
+          - node44
+          - node45
+      desc: NS masking on 1520 namespaces 76 subsystems and 5 initiators
+      destroy-cluster: false
+      module: test_ceph_nvmeof_ns_masking.py
+      name: Test E2E nvmeof namespace masking at scale
+      polarion-id: CEPH-83609782
+
+# Group 2 configuration  - In band, NS masking, set_QOS
+  - test:
+      abort-on-fail: true
+      config:
+        node: node13
+        rbd_pool: rbd2
+        do_not_create_image: true
+        rep-pool-only: true
+        steps:
+          - config:
+              service: subsystem
+              command: add
+              args:
+                subsystems: 76
+                max-namespaces: 2048
+          - config:
+              service: listener
+              command: add
+              args:
+                subsystems: 76
+                port: 4420
+                group: group2
+                nodes:
+                  - node13
+                  - node14
+                  - node15
+                  - node16
+                  - node17
+                  - node18
+                  - node19
+                  - node20
+          - config:
+              service: host
+              command: add
+              args:
+                subsystems: 76
+                group: group2
+      desc: GW group with 8 GWs and 76 subsystems
+      destroy-cluster: false
+      module: test_ceph_nvmeof_gateway_sub_scale.py
+      name: Configure 76 subsystems on group2
+      polarion-id: CEPH-83595512                     # This shall be replaced with In band authentication functionality at scale
+
+  - test:
+      abort-on-fail: true
+      config:
+        nodes:
+          - node13
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        service: namespace
+        steps:
+          - config:
+              command: add
+              args:
+                subsystems: 76
+                namespaces: 1520
+                pool: rbd2
+                image_size: 1T
+                no-auto-visible: true
+                group: group2
+        initiators:
+          - node41
+          - node42
+          - node43
+          - node44
+          - node45
+      desc: NS masking on 1520 namespaces 76 subsystems and 5 initiators
+      destroy-cluster: false
+      module: test_ceph_nvmeof_ns_masking.py
+      name: Test E2E nvmeof namespace masking at scale
+      polarion-id: CEPH-83609782             # Set QOS can be added after this test
+
+# Group 3 configuration - VMware uses cases
```

Comment:
Please make this self explainatory like `this acts as a  placeholder for manually running VMware use cases on a set of baremetal nodes assigned to group3, without requiring any prior configuration`.



Let me know if you want it more formal, technical, or casual.